### ISSUE TITLE
Autocorrect for Rails/LinkToBlank is wrong for link_to with parentheses

### ISSUE THREAD

**Issue by kentarohonda** [rubocop-hq/rubocop#6737]
Auto-correction of `Rails/LinkToBlank` produces syntactically incorrect code when `link_to` is used with parentheses.

--------

## Expected behavior

`, rel: 'noopener'` is expected to be inserted before the closing parenthesis.

```ruby
link_to('name', 'http://example.com', target: '_blank', rel: 'noopener')
```

## Actual behavior

`, rel: 'noopener'` is inserted after the closing parenthesis.

```ruby
link_to('name', 'http://example.com', target: '_blank'), rel: 'noopener'
```

## Steps to reproduce the problem

Example file(`test.rb`)
```ruby
link_to('name', 'http://example.com', target: '_blank')
```
```shell
$ rubocop -R -a test.rb 
Inspecting 1 file
E

Offenses:

test.rb:1:39: C: [Corrected] Rails/LinkToBlank: Specify a :rel option containing noopener.
link_to('name', 'http://example.com', target: '_blank')
                                      ^^^^^^^^^^^^^^^^
test.rb:1:56: E: Lint/Syntax: unexpected token tCOMMA
(Using Ruby 2.2 parser; configure using TargetRubyVersion parameter, under AllCops)
link_to('name', 'http://example.com', target: '_blank'), rel: 'noopener'
                                                       ^

1 file inspected, 2 offenses detected, 1 offense corrected
```

## RuboCop version

```
$ rubocop -V
0.63.1 (using Parser 2.6.0.0, running on ruby 2.4.5 x86_64-linux)
```

**Comment by koic** [comment]
Thanks for the feedback. I opened a PR #6743.
